docs/UserGuide.md

```diff
@@ -104,31 +103,33 @@ Examples:
   Selects the 1st person in the results of the `find` command.
 
 #### Clearing all entries : `clear`
-Clears all entries from the address book.<br>
-Format: `clear`  
+Clears all entries from the task list.<br>
+Format: `clear` 
 
 #### Exiting the program : `exit`
 Exits the program.<br>
 Format: `exit`  
 
 #### Saving the data 
-Address book data are saved in the hard disk automatically after any command that changes the data.<br>
+Task List data are saved in the hard disk automatically after any command that changes the data.<br>
 There is no need to save manually.
 
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
 **A**: Install the app in the other computer and overwrite the empty data file it creates with 
-       the file that contains the data of your previous Address Book folder.
+       the file that contains the data of your previous Tasks folder.
        
```

Comment:
In future, you might want to use partial screenshots capturing only the part in concern.